### ISSUE TITLE
`noDuplicateClassMember`, no-dupe-class-members

### ISSUE THREAD

**Issue by Conaclos** [eslint/eslint#16734]
### Rule details

https://eslint.org/docs/latest/rules/no-dupe-class-members

### Related ECMAScript feature

Classes, getter, setters, accessors, methods, class properties

### What type of rule is this?

Warns about a potential problem

### Example code

```js
See https://eslint.org/docs/latest/rules/no-dupe-class-members
```

### Why should this rule be in the core instead of a plugin?

It is part of ESLint's core rule and is similar to already implemented Rome rules (e.g. `noDuplicateObjectKeys`).

### Participation

- [X] I am willing to submit a pull request to implement this rule.

**Comment by Conaclos** [comment]
Sorry, It was an error (wrong project)
